### DISCUSSION TITLE
How to set a common prefix for multiple routes

### DISCUSSION THREAD

**Discussion by chiichen** [tokio-rs/axum#2590] [Q&A]
### Summary

There are two route paths
```
/api/v1/hello
/api/v1/world
```
In `Gin`(a web framework of Go), it can be represented as
```go
	router := gin.Default()
	v1 := router.Group("api/v1")
	{ 	
		v1.POST("/hello", HelloHandler)
		v1.POST("/world", WorldHandler)
	}
```

We can use router group to simplify the route path.I'm wondering if there is any similar approach in axum

### axum version

0.7.4

**Comment by dayvejones** [comment]
```
Router::new()
    .route("/api/v1/hello", post(hello_handler))
    .route("/api/v1/world", post(world_handler))
```

also you can use https://docs.rs/axum/latest/axum/routing/struct.Router.html#method.nest if you have many routes that would be convenient to group.
